packages/node/src/types.ts

```diff
@@ -260,8 +260,8 @@ export type EVMRequestCreatedLog = EVMTemplateRequestCreatedLog | EVMFullApiRequ
 
 export interface EVMRequestFulfilledLog extends EVMEventLogMetadata {
   readonly parsedLog:
-  | AirnodeLogDescription<AirnodeRrpLog<'ClientRequestFulfilled'>>
-  | AirnodeLogDescription<AirnodeRrpLog<'ClientRequestFailed'>>;
+    | AirnodeLogDescription<AirnodeRrpLog<'ClientRequestFulfilled'>>
```

Comment:
How is this formatting change possbile? We should have prettier setup

Reply:
TBH I have no idea :smile: My IDE will format it wrongly for some reason (by wrongly I mean the state in which it was before) but pre-commit hook formats it correctly hence the change. I don't know how it could have been commited there without a check :man_shrugging: